docs/capabilities.md

```diff
@@ -288,7 +288,9 @@ CAP_SET_ISOLATION_LEVEL_VIA_SQL | Set to 'yes' to force Tableau to set the trans
 
 Capability | Description | Default | Recommended
 -|-|-|-
+CAP_CONNECT_NO_CUSTOM_SQL | Set to 'yes' to disable the custom sql option. | &ndash; | &ndash; 
 CAP_EXTRACT_ONLY | Set to 'yes' to perform queries on extracted data only. | &ndash; | &ndash;  
+CAP_EXTRACT_ONLY_REFRESH  | Set to 'yes' to enable extracted data to be refreshed. | &ndash; | &ndash;  
```

Comment:
Not sure if this should be in this PR or another, but we should add this capability to https://github.com/tableau/connector-plugin-sdk/blob/master/samples/scenarios/extract_only/sqlite_extract/manifest.xml